docs/user/science.rst

```diff
@@ -0,0 +1,142 @@
+
+.. dropdown:: 📜 Modern themes and classic PDF outputs
+
+    .. figure:: /img/screenshot_rtd_downloads.png
+        :align: right
+
+    Use the latest state-of-the-art themes for web and have PDFs and e-book formats automatically generated.
+
+    New themes are improving every day, and when you write documentation based on Jupyter Book and Sphinx, you will separate your contents and semantics from your presentation logic. This way, you can keep up with the latest theme updates or try new themes.
+
+    Another example of the benefits from separating content and presentation logic: Your documentation also transforms into printable books and eBooks.
+
+.. dropdown:: 📐 Widgets, widgets and more widgets
+
+    Design your science project's layout and components with widgets from a rich eco-system of Open Source extensions built for many purposes. Special widgets help users display and interact with graphs, maps and more. :external+jupyterbook:doc:`Several <content/components>` `extensions <https://sphinx-gallery.github.io/>`__ are built and invented by the science community.
+
+.. dropdown:: ⚙️ Automatic builds
+
+    Build and publish your project for every change made through Git (GitHub, GitLab, BitBucket etc). Preview changes via Pull Requests. Receive notifications when something is wrong. How does this work? Have a look at this video:
+
+    .. TODO: Where should this video live?
+
+    .. video:: https://overtag.dk/files/enable-pull-request-builders.mp4
+       :width: 500
+       :height: 300
+
+.. dropdown:: 💬 Collaboration and community
+
+    .. figure:: /img/screenshot_edit_on_github.png
+        :align: right
+
+    Science and academia have a big kinship with software developers: We ❤️ community. Our solutions and projects become better when we foster inclusivity and active participation. Read the Docs features easy access for readers to suggest changes via GitHub, GitLab, BitBucket etc. But not just any unqualified feedback. Instead, the *code* and all the tools are available for your community to forge qualified contributions.
+
+    Your readers can become your co-authors!
+
+    Discuss changes via Pull Request and track all changes in your project's version history.
+
+    Does this sound chaotic? Don't worry, the full ownership and permission handling remains in your hands: Project and organization owners govern what is released and who has access to approve and build changes.
+
+.. dropdown:: 🔎 Search Engine Optimization
+
+    Don't reinvent SEO: Have SEO handled jointly by the combined best-practices from Sphinx, its themes and Read the Docs' platform.
+
+.. dropdown:: 🌱 Grow your own solutions
+
+    The eco-system is Open Source and makes it accessible for anyone with Python skills to build their own extensions.
```

Comment:
Open source doesn't need proper capitalization here. Microsoft's styleguide has some guidance on the term:

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source


```suggestion
    The eco-system is open source and makes it accessible for anyone with Python skills to build their own extensions.
```